monai/transforms/spatial/array.py

```diff
@@ -3142,6 +3142,9 @@ class GridPatch(Transform):
         pad_mode: refer to NumpyPadMode and PytorchPadMode. If None, no padding will be applied. Defaults to ``"constant"``.
         pad_kwargs: other arguments for the `np.pad` or `torch.pad` function.
 
+    Returns:
+        MetaTensor: A MetaTensor consisting of a batch of all the patches with associated metadata
```

Comment:
Hi @drbeh, "the associated metadata" just mean the meta data from original data, not pointing to the corresponding patches, right?
```
from monai.transforms import LoadImage, GridPatch, EnsureChannelFirst

image_arr = LoadImage(image_only=True)(image_path)
image1 = EnsureChannelFirst()(image_arr)
image2 = GridPatch(patch_size=(32,32,3), num_patches=10)(image1)

print(image1.meta['spatial_shape'] is image2.meta['spatial_shape']) # True
```
Does `spatial_shape` need to be updated?

Reply:
Good catch, thanks!